src/Module.Server/ModuleData/dtv/dtv_characters.xml

```diff
@@ -3963,7 +3963,7 @@
       <skill id="IronFlesh" value="9" />
       <skill id="PowerStrike" value="9" />
```

Comment:
Noticed a lot of your inf units have high stats e.g. PS of 9 and even higher. 
Could you please look at the other units of similar rounds and re-adjust the stats. 
To come back to the mentioned stat of PS, Dwarfs and tincans have 7 PS.